src/plugins/dam/components/uploader/FileList.jsx

```diff
@@ -65,6 +55,34 @@ class FileList extends React.Component {
     }
   }
 
+  ensureActiveItemVisible() {
+    if (!this.activeFileItem) {
+      return
+    }
+
+    const currentClassName = this.damFileList.current.className;
```

Comment:
@digitalartcrew dont think you really need a separate variable here, you can just put it into the if in the loop. also - is `this.damFileList` ever not set yet by the time we get to this comparison? i know that `this.activeFileItem` is not ready the first time